doc/releases/release-notes-3.3.rst

```diff
@@ -111,6 +111,11 @@ Deprecated in this release
 
   NOTE: Only functions are marked as ``__deprecated``, type definitions are not.
 
+* STM32 Ethernet Mac address Kconfig related symbols (kconfig:option:`CONFIG_ETH_STM32_HAL_RANDOM_MAC`,
+  kconfig:option:`CONFIG_ETH_STM32_HAL_MAC4`, ...) have been deprecated in favor
+  of the use of zephyr generic device tree `local-mac-address`` and `zephyr,random-mac-address``
```

Comment:
missing 2x quote on both

Reply:
I'm not seeing double quotes on other Kconfig symbols in this file, but I agree there should be a discrepancy somewhere. Anyway: fixed.